### ISSUE TITLE
Zotero.delete_tags(tag_a[, tag …])

### ISSUE THREAD

**Issue by retorquere** [edtechhub/zotero-cli#67]
Delete one or more tags from your library

**Comment by a1diablo** [comment]
This is not implemented in the pyzotero. However, there is an API to do it, but it needs a If-Unmodified-Since-Version number as a request parameter. I'm not sure how to retrieve this version for the tag because it is not available in the tags GET request. @retorquere Do you have any pointers for that? Thanks!

**Comment by retorquere** [comment]
The docs aren't really clear but my guess would be that it's the [library version](https://www.zotero.org/support/dev/web_api/v3/syncing#version_numbers).

**Comment by a1diablo** [comment]
Yeah, right. But the problem is, that tags do not have a version specified. At least it is not retrieved in the tag objects.

**Comment by retorquere** [comment]
I understand, but the library as a whole does have a version. My guess is you can pass the library version here.
